src/date-picker/date-picker-dialog.jsx

```diff
@@ -41,18 +42,6 @@ const DatePickerDialog = React.createClass({
     muiTheme: React.PropTypes.object,
   },
 
-  getDefaultProps: function() {
```

Comment:
This is a breaking change for people how are directly importing `date-picker-dialog.jsx`